### PR TITLE
added signal for sending an email to a client when a token is assigned to him

### PR DESCRIPTION
**Pull request by vinyll**
Hand tested with the tci-admin app.

### CODE REVIEW

File: tcipostman/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    db.signals.post_save.connect(Postman.send_client_token,
+                                 sender=models.SingleToken)
+
+
+class Postman:
+    app = None
+    mail = None
+
+    @classmethod
+    def send_client_token(klass, sender, document, created, **kwargs):
```
Comment (Ydrasil) [inline tcipostman/__init__.py]: I don't see when `sender` and I don't get the convention's names of the variables

Comment (vinyll) [reply tcipostman/__init__.py]: Right.Here's an explanation of each variable:

- `klass`: convention name for representing the current class in a class method. However, after checking [`cls` is declared preferred conventional name](https://www.python.org/dev/peps/pep-0008/)
- `sender, document, created`: arguments [called by the `post_save.connect()`](http://nullege.com/codes/show/src@m@o@mongoengine-0.8.7@mongoengine@document.py/301/mongoengine.signals.post_save.send)
- `kwargs`: extra arguments. They are not required here however.

File: tcipostman/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+def init_app(app):
+    Postman.app = app
+    Postman.mail = Mail(app)
+    db.signals.post_save.connect(Postman.send_client_token,
+                                 sender=models.SingleToken)
+
+
+class Postman:
+    app = None
+    mail = None
```
Comment (Ydrasil) [inline tcipostman/__init__.py:15]: I don't get it when `app` and `mail` are correctly set up in the function `send_client_token` with a project that use Postman

Comment (vinyll) [reply tcipostman/__init__.py:15]: Is [the setup in readme](https://github.com/anthropedia/tci-postman/pull/1/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R18) replying to your point?

**Comment by vinyll** [comment]
This is blocking for publishing. I'll merge and we create an issue if it should be subject to improvment

**Pull request merged**